### PR TITLE
Add Cmdlet Prefixes

### PR DESCRIPTION
**Pull request by doug-reimer**
Fixes issue #76.

Added Polaris as a prefix to commands:

i.e. New-WebRoute becomes New-PolarisWebRoute

### CODE REVIEW

**Comment by tylerl0706** [comment]
Awesome! Thanks for this. I'll merge this in as soon as I get https://github.com/PowerShell/Polaris/pull/77 merged in since it's a rather lengthy PR

**Comment by TimCurwick** [comment]
I recommend changing the nouns to be more concise while we are at it.

New-PolarisRoute
New-PolarisMiddleware

**Comment by tylerl0706** [comment]
@doug-reimer I just merged in a big PR by @TimCurwick (thank for that Tim!)  who added quite a bit of Pester tests that all use the old naming convention.

These will also need to be updated with this PR.

I would be incredibly thankful (get it? Because it's Thanksgiving) for that update - otherwise, I'm happy to address it when I get back from vacation!

Thanks a lot for contributing!

**Comment by doug-reimer** [comment]
I'll get those updated.

**Comment by doug-reimer** [comment]
I modified a couple of the Cmdlet names to improve conciseness as per @TimCurwick suggestion.

File: Polaris.psm1 [hunk 1]
```diff
@@ -1,609 +1,836 @@
...
+.EXAMPLE
+    Remove-PolarisRouteMiddleware -Name JsonBodyParser
+.EXAMPLE
+    Remove-PolarisRouteMiddleware
+    Removes all route middleware.
+.EXAMPLE
+    Remove-PolarisRouteMiddleware -Name ParamCheck*, ParamVerify*
+    Removes any route middleware with names starting with ParamCheck or ParamVerify.
+.EXAMPLE
+    Get-RouteMiddelware | Remove-PolarisRouteMiddleware
```
Comment (tylerl0706) [inline Polaris.psm1]: Nit: fix example

**Comment by doug-reimer** [comment]
I have fixed the typo in the example. Is keeping the reference to "route" in middleware important to identifying the "type" of middleware?

**Comment by tylerl0706** [comment]
Very good point. We may want to have a different type of middleware in the future.

Ill merge this in. Thanks for contributing!

**Comment by doug-reimer** [comment]
I guess some of the wiki documentation will also have to be updated to reflect the the Cmdlet names.

**Comment by chriskuech** [comment]
This can actually be done with a single config line change is the module manifest: `DefaultCommandPrefix = 'Polaris'`

The test compatibility could hypothetically be fixed with `Import-Module Polaris -Prefix ''` in the tests, though I haven't tested it to verify.

**Pull request merged**